build_tools/travis/install.sh

```diff
@@ -53,11 +53,11 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl
+            cython=$CYTHON_VERSION libgfortran mkl flake8
```

Comment:
I think the concern here is that if flake8, now or in the future, has other dependencies, then we are installing dependencies that may give us false green CI icons in the future. Seems unlikely, but I don't think it's a great hassle to set this up separately.